src/test/resources/repoconfig_merge_test/repo-config.csv

```diff
@@ -1,2 +1,2 @@
-Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config
-https://github.com/reposense/testrepo-Beta.git,master,,collated**,
+Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config,Ignore Commits List
+https://github.com/reposense/testrepo-Beta.git,master,,collated**,,
```

Comment:
also, the csv wouldn't be parsed if the column numbers do not match?

Reply:
Yes